core/types/authorization.go

```diff
@@ -75,7 +77,7 @@ func (ath *Authorization) RecoverSigner(data *bytes.Buffer, b []byte) (*libcommo
 		return nil, fmt.Errorf("invalid v value: %d", ath.V.Uint64())
 	}
 
-	if !crypto.ValidateSignatureValues(sig[64], &ath.R, &ath.S, false) {
+	if !libcrypto.TransactionSignatureIsValid(sig[64], &ath.R, &ath.S, false /* allowPreEip2s */) {
```

Comment:
in this PR- https://github.com/erigontech/erigon/pull/11952
I changed false -> true, which would allow Secp256k1halfN check on s

Reply:
Right. This PR essentially does the same, because the meaning of the last argument is flipped in `TransactionSignatureIsValid` compared with `ValidateSignatureValues`.